src/Integration.Vsix/SonarLintTelemetryPackage.cs

```diff
@@ -53,7 +53,7 @@ private async System.Threading.Tasks.Task InitOnUIThreadAsync()
         {
             Debug.Assert(ThreadHelper.CheckAccess(), "Expecting to be on the UI thread");
 
-            var logger = await this.GetMefServiceAsync<ILogger>();
```

Comment:
Hmm, presumably we will then fail on initializing the next package? Perhaps it's worth adding try-catches to all of them? (although I do find these errors useful when developing and try-catching will hurt that).

Reply:
> Hmm, presumably we will then fail on initializing the next package?

Not necessarily. An exception in one package wouldn't have stopped VS from loading other packages when required, so if they were going to fail I would have expected errors for those too. We don't know what the underlying issue was; it's possible that later calls to fetch the logger would succeed. We just don't know.

try-catch in all packages: could do. About half of the packages do, the other half don't. The main reason for adding it is cosmetic, to avoid a modal dialogue with SLVS in the call stack. Our stuff still won't work.

I'll commit this as-is and look at the other bug, which also seems to be package-related, to see if there is a pattern.